### ISSUE TITLE
Anterior shoulder pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10811]
**Preferred term label:**
Anterior shoulder pain

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain localized to the anterior aspect of the shoulder

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030834

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Proximal biceps tendon tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001141
- label: Anterior shoulder pain
- parents: [Shoulder pain (HP:0030834)](https://hpo.jax.org/app/browse/term/HP:0030834)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the anterior (front) part of the shoulder.
- comment: Pain localized to the anterior aspect of the shoulder. Anterior shoulder pain may be observed in conditions such as proximal biceps tendon tear.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10811
- synonyms:
